Constitution.md

```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
 
 ## Article 2b: Executive Officers and Duties
 
-### President(s)
+
+### Co-Presidents
+
+The Co-presidents is a transitional role approved by the 2022-2023 council for the 2023-2024 CSSA general elections and elected as a single voting member on council.
 
 1. Act as a signing authority for CSSA.
 2. Be the contact person for UMSU.
-3. Attend Student Executive Assembly (SEA) Meetings on behalf of CSSA.
-4. Be responsible for the overall operations of the group.
-5. Deal with issues concerning the club and its individual Members.
-6. Serve as signing authority on any group accounts.
-7. Be a voting member of CSSA for management decisions.
-8. Call special meetings.
-9. Attend all meetings.
-10. Carry out provisions of the constitution.
-11. Perform any other duties incidental to the office of the President or directed by Council.
+3. Maintain frequent and punctual communication with the Faculty of Science, Department of Computer Science, UMSU, Members of Council, Members of the Association, and all other relevant parties.
+4. Attend Student Executive Assembly (SEA) Meetings on behalf of CSSA.
+5. Be responsible for the overall operations of the group.
+6. Deal with issues concerning the club and its individual Members.
+7. Serve as signing authority on any group accounts.
+8. Be a voting member of CSSA for management decisions.
+9. Call special meetings.
+10. Attend all meetings.
```

Comment:
We need to define meetings because I don't think we want this once we have committees.